### DISCUSSION TITLE
Update File in Users Table

### DISCUSSION THREAD

**Discussion by milimyname** [pocketbase/pocketbase#3207] [Q&A] [answered]
Hi everyone.

I wonder if I can update a file in the users table?

```js
const formData = new FormData();
formData.append('avatar', blob);

await pocketbase.collection('users').update(user_id, {
      avatar: formData  // formData.get('avatar')
});
```

It just puts N/A. Is there any workaround?

**Comment by ganigeorgiev** [comment, marked as answer]
The above syntax is incorrect. You are mixing the `FormData` and plain object body format. You have to use only one of them, aka. try with:

```js
const formData = new FormData();
formData.append('avatar', blob);
await pocketbase.collection('users').update(user_id, formData);

// or with JS SDK v0.17+ the below is also supported
await pocketbase.collection('users').update(user_id, { avatar: blob });
```

**Reply by milimyname** [reply]
Thanks!  Below worked for me.
`await pocketbase.collection('users').update(user_id, formData);`

**Reply by ganigeorgiev** [reply]
Both should work, but the latter is supported only with JS SDK v0.17+.
_Note that in your example it is incorrect because you are assigning to "avatar" the `formData` variable instead of directly the `blob` var._
